app/helpers/application_helper/toolbar/container_center.rb

```diff
@@ -11,16 +11,19 @@ class ApplicationHelper::Toolbar::ContainerCenter < ApplicationHelper::Toolbar::
           'ff ff-timeline fa-lg',
           N_('Show Timelines for this Container'),
           N_('Timelines'),
+          :url       => "/show",
           :url_parms => "?display=timeline",
           :options   => {:entity => 'Container'},
-          :klass     => ApplicationHelper::Button::ContainerTimeline),
+          :klass     => ApplicationHelper::Button::ContainerTimeline
+        ),
         button(
           :container_perf,
           'ff ff-monitoring fa-lg',
           N_('Show Capacity & Utilization data for this Container'),
           N_('Utilization'),
-          :url_parms => "?display=performance",
-          :klass     => ApplicationHelper::Button::VmPerf),
+          :url       => "/show",
+          :url_parms => "?display=performance"
```

Comment:
@yaacov we want this button to be disabled if there is no data like the in [vms](https://github.com/manageiq/manageiq-ui-classic/blob/ce8b2170b2ab790aa45d60e32a76659d4c92ac64/app/helpers/application_helper/button/vm_perf.rb#L5)

`has_perf_data?` does not work for containers?

Reply:
> @yaacov we want this button to be disabled if there is no data like in vms

We can not use the vms button `ApplicationHelper::Button::VmPerf` because it show the message:
"No C&U data has been collected for this **VM**" and container is not VM.

> we want this button to be disabled if there is no data 

In all other container objects, we show the button, and only after we get into the page we get the "no data message", this is the behaviour for container -> pods [1] and contianer -> nodes [2] . Container should have the same behaviour.

[1] https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/application_helper/toolbar/container_group_center.rb#L21
[2] https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/application_helper/toolbar/container_node_center.rb#L21

Reply:
If all this meant to fix is the message, can you just update the message?

Either copy `VmPerf` to `ContainerPerf` and update the message, or add a field to the definition which you can read in `disabled?`.

(But keep the message whole, even though you need 2 copies that way :).)

Reply:
> If all this meant to fix is the message,

No, the main task of this PR is to:
a. prevent the graying out of the "Utilization" button in the drop down.
b. make this button work :-) and send us to the Utilization page.
c. the "VM" message is not mentioned in the BZ, but we should fix it as well.

This behaviour (not graying out) is consistent with the behaviour of this button in the providers, nodes and pods pages, and also with the behaviour of the "Timeline" button in providers.

We opened a new BZ about making **all** the buttons consistent [1] but this is not a priority.

[1]  https://bugzilla.redhat.com/show_bug.cgi?id=1512276